db/migrate/20241105175000_add_life_cycle_to_work_packages.rb

```diff
@@ -0,0 +1,8 @@
+class AddLifeCycleToWorkPackages < ActiveRecord::Migration[7.1]
+  def change
+    add_reference :work_packages,
+                  :project_life_cycle,
+                  foreign_key: { to_table: :project_life_cycles },
```

Comment:
I think here rails can guess the table, so can be:
```suggestion
                  foreign_key: true,
```